helm/kubernetes-kube-state-metrics-chart/templates/psp.yaml

```diff
@@ -6,21 +6,26 @@ metadata:
     app: {{ .Values.name }}
     giantswarm.io/service-type: "managed"
 spec:
-  allowPrivilegeEscalation: true
-  fsGroup:
-    rule: RunAsAny
-  privileged: true
+  privileged: false
   runAsUser:
-    rule: RunAsAny
+    rule: MustRunAsNonRoot
   seLinux:
```

Comment:
I have decided to use the same psp configuration as upstream as it way more secure